core/Error.h

```diff
@@ -157,9 +157,10 @@ class ErrorBuilder {
     }
 
     void addAutocorrect(AutocorrectSuggestion &&autocorrect);
-    template <typename... Args> void replaceWith(Loc loc, const std::string &msg, const Args &... args) {
-        std::string formatted = fmt::format(msg, args...);
-        addAutocorrect(AutocorrectSuggestion(loc, move(formatted)));
+    template <typename... Args>
+    void replaceWith(const std::string message, Loc loc, const std::string &replacement, const Args &... args) {
```

Comment:
`message` seems too general. `Title`? `buttonName`?

Reply:
`title` wfm

Reply:
also is it markdown? if so `titleMarkdown`?

Reply:
nope, it's plaintext unfortunately

Reply:
![image](https://user-images.githubusercontent.com/4211838/62397089-ac44fe00-b529-11e9-9ef0-3ea3277ab61d.png)

I kept the backticks in since I think it makes the message clearer but I can remove them.